makefile

```diff
@@ -3,7 +3,7 @@ GENDIR := gen
 GOPATH_GENDIR := $(GOPATH_DIR)/$(GENDIR)
 
 # Find all .proto files.
-PROTO_FILES := $(wildcard opentelemetry/proto/*/v1/*.proto opentelemetry/proto/collector/*/v1/*.proto opentelemetry/proto/experimental/*/*.proto)
+PROTO_FILES := $(wildcard opentelemetry/proto/*/v1/*.proto opentelemetry/proto/*/experimental/*.proto opentelemetry/proto/collector/*/v1/*.proto)
```

Comment:
PROTO_FILES := $(wildcard opentelemetry/proto/*/*/*.proto opentelemetry/proto/*/*/*/*.proto)?

Reply:
Yeah! That's good idea. Updated